apps/desktop-wallet/src/api/addressQueries.ts

```diff
@@ -64,6 +71,11 @@ export const addressAlphBalanceQuery = ({
   })
 }
 
+export type AddressTokensBalancesQueryFnData = {
+  addressHash: AddressHash
+  tokenBalances: TokenDisplayBalances[]
+}
+
```

Comment:
Defining return types of queries will improve readability of our code, especially when we come back to it after some time while working on other stuff